File: html/BeautyLanding.java
import javax.swing.*;
import java.awt.*;

public class BeautyLanding extends JFrame {
    public BeautyLanding() {
        setTitle("LookList – Luxury Beauty Marketplace");
        setSize(600, 400);
        setDefaultCloseOperation(JFrame.EXIT_ON_CLOSE);
        setLocationRelativeTo(null);

        // Custom panel for background and font
        JPanel panel = new JPanel() {
            @Override
            protected void paintComponent(Graphics g) {
                super.paintComponent(g);
                // Gradient background
                Graphics2D g2d = (Graphics2D) g;
                GradientPaint gp = new GradientPaint(0, 0, new Color(24,18,43), getWidth(), getHeight(), new Color(255,56,100));
                g2d.setPaint(gp);
                g2d.fillRect(0, 0, getWidth(), getHeight());
            }
        };
        panel.setLayout(new BoxLayout(panel, BoxLayout.Y_AXIS));

        // Use Google Fonts like Poppins or Inter if installed, fallback otherwise
        Font headerFont = new Font("Poppins", Font.BOLD, 32);
        Font descFont = new Font("Inter", Font.PLAIN, 18);

        JLabel title = new JLabel("LookList");
        title.setFont(headerFont);
        title.setForeground(new Color(192,192,192));
        title.setAlignmentX(Component.CENTER_ALIGNMENT);

        JLabel desc = new JLabel("<html><div style='text-align: center;'>Discover, book, and pay for <span style='color:#ff3864;'>premium beauty services</span> near you — hair, nails, henna, makeup, and more.</div></html>");
        desc.setFont(descFont);
        desc.setForeground(Color.WHITE);
        desc.setAlignmentX(Component.CENTER_ALIGNMENT);

        JButton ctaBtn = new JButton("Search Near Me");
        ctaBtn.setFont(headerFont.deriveFont(18f));
        ctaBtn.setForeground(Color.WHITE);
        ctaBtn.setBackground(new Color(255,56,100));
        ctaBtn.setAlignmentX(Component.CENTER_ALIGNMENT);
        ctaBtn.setFocusPainted(false);

        panel.add(Box.createVerticalGlue());
        panel.add(title);
        panel.add(Box.createRigidArea(new Dimension(0, 20)));
        panel.add(desc);
        panel.add(Box.createRigidArea(new Dimension(0, 30)));
        panel.add(ctaBtn);
        panel.add(Box.createVerticalGlue());

        add(panel);
    }

    public static void main(String[] args) {
        // Set Look and Feel to system default
        try { UIManager.setLookAndFeel(UIManager.getSystemLookAndFeelClassName()); } catch (Exception e) {}
        SwingUtilities.invokeLater(() -> new BeautyLanding().setVisible(true));
    }
}

File: html/beauty_landing.cpp
// To compile: g++ beauty_landing.cpp -o beauty_landing -lsfml-graphics -lsfml-window -lsfml-system
#include <SFML/Graphics.hpp>

int main() {
    sf::RenderWindow window(sf::VideoMode(600, 400), "LookList – Luxury Beauty Marketplace");

    // Gradient background emulation (linear interpolation)
    sf::Image bgImage;
    bgImage.create(600, 400);
    for (unsigned y = 0; y < 400; ++y) {
        for (unsigned x = 0; x < 600; ++x) {
            float t = float(y) / 400.0f;
            sf::Color c(
                static_cast<sf::Uint8>(24 + t * (255 - 24)),
                static_cast<sf::Uint8>(18 + t * (56 - 18)),
                static_cast<sf::Uint8>(43 + t * (100 - 43))
            );
            bgImage.setPixel(x, y, c);
        }
    }
    sf::Texture bgTexture;
    bgTexture.loadFromImage(bgImage);
    sf::Sprite bgSprite(bgTexture);

    // Load a custom font (put "Poppins-Bold.ttf" in the same directory, or use any .ttf)
    sf::Font font;
    if (!font.loadFromFile("Poppins-Bold.ttf")) {
        // Fallback to default font, or handle error
        return 1;
    }

    sf::Text title("LookList", font, 48);
    title.setFillColor(sf::Color(192,192,192));
    title.setPosition(180, 40);

    sf::Text desc("Discover, book, and pay for premium beauty services\nnear you — hair, nails, henna, makeup, and more.", font, 20);
    desc.setFillColor(sf::Color::White);
    desc.setPosition(60, 120);

    sf::RectangleShape button(sf::Vector2f(220, 56));
    button.setPosition(190, 250);
    button.setFillColor(sf::Color(255,56,100));
    button.setOutlineThickness(2);
    button.setOutlineColor(sf::Color(201,24,74));

    sf::Text btnText("Search Near Me", font, 24);
    btnText.setFillColor(sf::Color::White);
    btnText.setPosition(202, 260);

    while (window.isOpen()) {
        sf::Event event;
        while (window.pollEvent(event)) {
            if (event.type == sf::Event::Closed)
                window.close();
        }
        window.clear();
        window.draw(bgSprite);
        window.draw(title);
        window.draw(desc);
        window.draw(button);
        window.draw(btnText);
        window.display();
    }
    return 0;
}